.circleci/config.yml

```diff
@@ -20,27 +20,55 @@ set_npm_auth: &set_npm_auth
 
 restore_dependency_cache_unix: &restore_dependency_cache_unix
   restore_cache:
+    name: Restore NPM Cache
     keys:
       - v9-cache-unix-{{ checksum "package-lock.json" }}
-      - v9-cache-unix-
+
+restore_build: &restore_build
+  restore_cache:
+    name: Restore Axe.js Cache
+    keys:
+      - v9-cache-build-<< pipeline.git.revision >>
 
 jobs:
   # Fetch and cache dependencies.
   dependencies_unix:
     <<: *defaults
     <<: *unix_box
     steps:
-      - browser-tools/install-browser-tools
       - checkout
-      - <<: *set_npm_auth
       - <<: *restore_dependency_cache_unix
+      - run:
+          name: Skip Install If Cache Exists
+          command: |
+            if [ -d "node_modules" ]; then
+                echo "node_modules exist"
+                circleci step halt
+            else
+                echo "node_modules does not exist"
+            fi
+      - browser-tools/install-browser-tools
+      - <<: *set_npm_auth
       - run: npm ci
       - run: npx browser-driver-manager install chromedriver --verbose
       - save_cache:
           key: v9-cache-unix-{{ checksum "package-lock.json" }}
           paths:
             - node_modules
 
+  # Build and cache axe.js
+  build_unix:
+    <<: *defaults
+    <<: *unix_box
+    steps:
+      - checkout
+      - <<: *restore_dependency_cache_unix
+      - run: npm run build
+      - save_cache:
+          key: v9-cache-build-<< pipeline.git.revision >>
+          paths:
+            - axe.js
```

Comment:
You need to cache `axe.min.js` too, otherwise it won't be published. And then what about `metadata-function-map.js`? Might we need that cached?

Reply:
Good point. We don't need to cache the function map as it get's rolled into the built files.